pkg/config/types.go

```diff
@@ -120,3 +120,24 @@ func (q *BandwidthQuantity) MarshalJSON() ([]byte, error) {
 func (q *BandwidthQuantity) Bytes() int64 {
 	return q.i
 }
+
+type BandwidthLimitMode string
+
+const (
+	BandwidthLimitModeClient BandwidthLimitMode = "client"
+	BandwidthLimitModeServer BandwidthLimitMode = "server"
+)
+
+// Create a new BandwidthLimitMode from a string. Default to "client".
+func NewBandwidthLimitMode(s string) (BandwidthLimitMode, error) {
+	switch BandwidthLimitMode(s) {
+	case BandwidthLimitModeClient, BandwidthLimitModeServer:
+		return BandwidthLimitMode(s), nil
+	default:
+		return BandwidthLimitModeClient, errors.New("invalid bandwidth_limit_mode")
```

Comment:
returning an error here to help with validation when parsing from the config file.  will always default to `"client"` in these cases.

Reply:
this is now outdated